matrix-synapse-riot/terraform/matrix-server/bootstrap-dehydrated.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
Should this file be named `dehydrated-bootstrap` for consistency with `dehydrated-renew`?

They are actually the same file, why not just reference one copy?

Reply:
I am keeping it named bootstrap-dehydrated.sh because its being called from main.tf and now there is only one copy of the file.